docs/ioformats.rst

```diff
@@ -95,24 +95,23 @@ Download and read from URL:
 Write out audio files
 ---------------------
 
-*librosa* uses `scipy.io.wavfile <https://docs.scipy.org/doc/scipy/reference/generated/scipy.io.wavfile.write.html>`_ to write out wav files. Please be aware, that this function uses the numpy dtype to determine the PCM subtype. For example if your processed audio array is of dtype ``np.float64`` (which is the default on most machines), your resulting wav file would be of type 64bit float as well. This is not considered to be a `standard PCM wavfile <https://msdn.microsoft.com/en-us/library/windows/hardware/dn653308%28v=vs.85%29.aspx>`_. If you would like to write 16bit PCM you could convert your array before hand:
+*librosa* provides a thin wrapper around `scipy.io.wavfile <https://docs.scipy.org/doc/scipy/reference/generated/scipy.io.wavfile.write.html>`_ to write out wav files. 
 
 .. code-block:: python
     :linenos:
 
     import numpy as np
-    import librosa
 
     rate = 44100
-    audio = np.random.uniform(-1, 1, size = (rate * 10, 2))
+    data = np.random.randn(2 * rate)
+
+    librosa.output.write_wav('file.wav', data, rate)
+
 
-    maxv = np.iinfo(np.int16).max
-    librosa.output.write_wav(
-        "out_int16.wav", (audio * maxv).astype(np.int16), rate
-    )
 
-Writing audio files using pysoundfile is similar to the method in *librosa*, however it can automatically
-convert to a given PCM subtype and additionally support several compressed formats like *FLAC* or *OGG*:
+Please be aware that this function only supports floating-point inputs. For example if your processed audio array is of dtype ``np.float64`` (which is the default on most machines), your resulting wav file would be of type 64bit float as well. This is not considered to be a `standard PCM wavfile <https://msdn.microsoft.com/en-us/library/windows/hardware/dn653308%28v=vs.85%29.aspx>`_, though most wav readers should be able to load it without problems.
```

Comment:
Missing comma after "For example"
64bit -> 64-bit
wav -> WAV
though -> although (more prevalent within non-native En speakers)

Perhaps it would be good to say that the most common PCM standard is 16-bit integer?